.env.example

```diff
@@ -1,2 +1,5 @@
 MPT_3500_GITHUB_KEY="your-client-id"
 MPT_3500_GITHUB_SECRET="your-client-secret"
+SMTP_PASSWORD='your-mandrill-api-key'
+SMTP_USERNAME='your-mandrill-email'
+DEFAULT_URL_HOST='localhost:3000'
```

Comment:
these 2 env vars will be required for prod -- can someone help me create a fancy user-provided service for them?